src/app/v2/inventory/projects/[projectId]/actions.ts

```diff
@@ -0,0 +1,37 @@
+"use server";
+import { createProjectListClient } from "@/lib/rpc/client";
+import { sessionMustGetTenant } from "@/lib/session/session";
+import { getAccessToken } from "@auth0/nextjs-auth0";
+
+export async function listProjectVersions(projectId: string) {
+  const { accessToken } = await getAccessToken();
+  if (!accessToken) {
+    throw new Error("No access token found");
+  }
+  const tenant = await sessionMustGetTenant();
+  const service = createProjectListClient(tenant, accessToken);
+  return await service.listProjectVersions({ projectId });
+}
+
+export async function listProjectVersionBOM(versionId: string) {
+  const { accessToken } = await getAccessToken();
+  if (!accessToken) {
+    throw new Error("No access token found");
+  }
+  const tenant = await sessionMustGetTenant();
+  const service = createProjectListClient(tenant, accessToken);
```

Comment:
```suggestion
  const service = createProjectServiceClient(tenant, accessToken);
```

See the PR comment here:
https://github.com/safedep/platform.safedep.io/pull/66#discussion_r1931572873